azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -255,8 +255,9 @@ public String call() throws Exception {
           // max 5 secs
           String jsonString = refreshPair.getSecond().get(5, TimeUnit.SECONDS);
           executor.setExecutorInfo(ExecutorInfo.fromJSONString(jsonString));
-          logger.info("Successfully refreshed ExecutorInfo for executor: "
-            + executor);
+          logger.info(String.format(
```

Comment:
How big is the jsonString?  Do we really need to see the jsonString? or may be log out jsonString in a separate statement with debug level.